moped-editor/src/views/projects/projectsListView/ProjectsListViewTable.js

```diff
@@ -292,6 +296,11 @@ const ProjectsListViewTable = ({ title, query, searchTerm, referenceData }) => {
           {entry.project_name}
         </RouterLink>
       ),
+      cellStyle: {
```

Comment:
There is another cellStyle set on line 285 that is causing a warning that Netlify doesn't like, combine these and this is good to go!

Reply:
thank you! i didnt review again after merging main 😨

Reply:
All good! 🚀

Reply:
totally slipped my mind that were workin on the same part of the codebase 😅